maintainers/working_groups/learning_journey/documentation-survey.md

```diff
@@ -61,6 +61,14 @@ To better navigate the material and judge its relevance, every entry should prov
 - https://nixos.org/guides/nix-pills/generic-builders.html
 - https://nixos.org/guides/nix-pills/automatic-runtime-dependencies.html
 - https://nixos.org/guides/nix-pills/developing-with-nix-shell.html
+
+- [Nix Pills Chapter 10. Developing with nix-shell](https://nixos.org/guides/nix-pills/developing-with-nix-shell.html)
+  nix-shell and integration of shell with derivation, `nix-shell`, `derivation`.   
```

Comment:
I'm not sure what that means, could you clarify?

Reply:
I'll try: 
by nix-shell, I mean the nix-shell command. 
I guess it's redundant to mention though it as it's listed as a covered command later in the sentence.

By integration with derivation - I mean getting the derivation creating part  (specifically the autotools.nix file in this example) to support the `nix-shell` command in addition to building the derivation with `nix-build`.

Reply:
Maybe something like: making a development environment from a derivation?